js/kadai-3.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function(e){
+  let id = 0;
+  document.querySelector("#task_add_button").addEventListener('click',function(){
+    let task_text = document.getElementById("task_text").value
+    // console.log(task_text);
+    if(task_text){
+      // console.log(task_text);
+      let tbody = document.getElementById("task_list")
+      // console.log(tbody)
+      let row = tbody.insertRow(-1)
+      let td_id = row.insertCell();
+      let td_comment = row.insertCell();
+      let td_state_button = row.insertCell();
+      let td_delete_button = row.insertCell();
+      let state_button = document.createElement("button");
+      let delete_button = document.createElement("button");
+      state_button.setAttribute("value","working");
```

Comment:
`,`の後ろにはスペースを空けましょう。
また、文字列は`''`にしましょう。
`""`は慣習として使用されません。

```suggestion
      state_button.setAttribute('value', 'working');
```